### DISCUSSION TITLE
Convert PDF to PDF/A, or to leave it as it is?

### DISCUSSION THREAD

**Discussion by greybit** [sumatrapdfreader/sumatrapdf#3468] [Q&A] [answered]
Hallo,

my question has nothing to do with Sumatra. Its more about PDF and the PDF/A specification.

The last weeks i was into converting my existing PDF files to PDF/A. A lot of documents from diffrent sources (all kinds of PDF versions, creators and some are signed).
I run in a lot of trouble and i asked myself if converting PDF/A is really a good idea? Even the programs converting to PDF/A are often divided in what is PDF/A comply with and what is not.
If you search at google you find a lot of articles the recommend converting to PDF/A, but these articles are often written from companys that sell this kind of products, so they are not very neutral.

I would like to know from a technical point of view if it is better to convert normal PDF to PDF/A, or to leave it as it is?

Regards, Michael

**Comment by GitHubRulesOK** [comment]
In My Humble Biased Opinion  

It depends, as PDF A is a Standard for Archiving (with different levels) so only of value to those publishing as archives.
the Point about PDF/A-1 is its SAFE legacy (few enhancements allowed except most basic for accessibility)
whereas PDF/A/2/3/4 are able to carry other commercial financial exploits to varying increasing degrees and require more humans for checking compliance.

**Comment by greybit** [comment]
Thank you for your answer.

I allready know about the diffrent PDF/A versions and levels with there there pros und cons. That wasn't what I meant. Btw, sorry for my english.

Ok, what you see often if you convert PDF to PDF/A is that LZW was converted to ZIP (for example). This LZW program code exists inside PDF readers. I did some research and found out that there was a patent on LZW but it expired long ago. So i dont understand if pdf readers one day can drop support for LZW, or any other program code so that I can no longer open my PDF files, for example in 20-30 years? This is my question: Is it realistic at all that these functions will disappear from the pdf reader and then I can no longer open my file?

**Reply by GitHubRulesOK** [reply, marked as answer]
I dont think anyone can guess file formats in 20 years time, PDF/A is an atempt to "Baseline" for the future blind readers and many older farmats like WordPerfect are hard to find readers for today but last century they were initially more common than MS Word docs (which often don't work in modern MSWord) By far the best long term are those you could read in Edlin (1980; 43 years ago) which are still perfectly readable in any console editor today, like say MS NotePad, Write WordPad Excel PowerPoint and SumatraPDF  plus thousands of other apps. but its not a nice commercial money spinner and was dropped decades ago.
